README.md

```diff
@@ -0,0 +1,38 @@
+# OpenJDK Developer's Guide
+
+This project maintains the [OpenJDK Developers' Guide](https://openjdk.java.net/guide/).
+The goal of this guide is to answer questions that the developers of the JDK might have around
+development process, tooling, standards, etc. The formal rules and processes are described in
+other documents, like [JEP 1](https://openjdk.java.net/jeps/1) for the JDK Enhancement-Proposal
+& Roadmap Process, or [JEP 3](https://openjdk.java.net/jeps/3) for the JDK Release Process.
+This guide is meant to be a complement with tutorials and examples for how to follow these rules
+and how to work together with the rest of the OpenJDK community.
+
+There are many common use cases that aren't detailed in the formal process. This guide contains the
+defacto standard for how to work in such cases.
+
+## Audience
+
+The target audience for this document is anyone in the OpenJDK community who aims to contribute
+to the development of OpenJDK. The nature of such a document is to target those who are not
+familiar with the process. People who are regular contributors would already know much of
+what this guide has to offer. Still, the guide should work as a source of knowledge also for
+experienced contributors. This means that any descriptions in the guide should be self-contained
+or have explicit references to any information the reader is expected to already know. The information
+should also be structured in such a way that it's easy to find details for any process, so that
+one who already knows the big picture can quickly find that particular detail that was forgotten.
+
+## Contributing
+
+To engage in the development of the guide itself create a private fork of the guide and join
```

Comment:
I would capitalize "Guide" when used as a shortened form of the "Developers' Guide."